packages/pretty-format/src/collections.ts

```diff
@@ -176,7 +176,7 @@ export function printObjectProperties(
   printer: Printer,
 ): string {
   let result = '';
-  const keys = getKeysOfEnumerableProperties(val);
+  const keys = getKeysOfEnumerableProperties(val) as Array<string>;
```

Comment:
does this need to be casted, provided that it's already done in line 22? Maybe declaring the return type for `getKeysOfEnumerableProperties` would help

Reply:
Whoops first added it here, then decided to change it in getKeysOfEnumerableProperties, and forgot to take this one off. Thanks!, BTW- this functions is duplicated a few times throughout, maybe we should move this in `jest-utils` in different PR?